src/app/frontend/deploy/createnamespace_controller.js

```diff
@@ -74,8 +77,9 @@ export default class NamespaceDialogController {
           this.mdDialog_.hide(this.namespace);
         },
         (err) => {
-          this.log_.info('Error creating namespace:', err);
           this.mdDialog_.hide();
+          this.errorDialog_.open('Error creating namespace', err);
```

Comment:
This is why we need type information. You called errorDialog_.open with second parameter, but the method does not accept it. With type information compiler would complain :)